### PR TITLE
Add support for video params

### PR DESCRIPTION
**Pull request by shubhamc-ins**

### CODE REVIEW

File: modules/insticatorBidAdapter.js [hunk 1]
```diff
@@ -12,6 +12,35 @@ const USER_ID_COOKIE_EXP = 2592000000; // 30 days
...
+  'sequence': (value) => isInteger(value),
+  'battr': (value) => isArrayOfNums(value),
+  'maxextended': (value) => isInteger(value),
+  'minbitrate': (value) => isInteger(value),
+  'maxbitrate': (value) => isInteger(value),
+  'playbackmethod': (value) => isArrayOfNums(value),
+  'playbackend': (value) => isInteger(value),
+  'delivery': (value) => isArrayOfNums(value),
+  'pos': (value) => isInteger(value),
+  'api': (value) => isArrayOfNums(value)
```
Comment (rohanInsticator) [inline modules/insticatorBidAdapter.js]: even this field should be limited to array values we support like protocols

File: modules/insticatorBidAdapter.js [hunk 2]
```diff
@@ -12,6 +12,35 @@ const USER_ID_COOKIE_EXP = 2592000000; // 30 days
...
+  'minduration': (value) => isInteger(value),
+  'maxduration': (value) => isInteger(value),
+  'protocols': (value) => isSubarray(value, [2, 3, 5, 6, 7, 8]), // protocols values supported by Inticator, according to the OpenRTB spec
+  'startdelay': (value) => isInteger(value),
+  'linearity': (value) => [1, 2].includes(value),
+  'skip': (value) => [1, 0].includes(value),
+  'skipmin': (value) => isInteger(value),
+  'skipafter': (value) => isInteger(value),
+  'sequence': (value) => isInteger(value),
+  'battr': (value) => isArrayOfNums(value),
```
Comment (rohanInsticator) [inline modules/insticatorBidAdapter.js]: same as api & protocols

File: modules/insticatorBidAdapter.js [hunk 3]
```diff
@@ -12,6 +12,35 @@ const USER_ID_COOKIE_EXP = 2592000000; // 30 days
...
+  'maxextended': (value) => isInteger(value),
+  'minbitrate': (value) => isInteger(value),
+  'maxbitrate': (value) => isInteger(value),
+  'playbackmethod': (value) => isArrayOfNums(value),
```
Comment (rohanInsticator) [inline modules/insticatorBidAdapter.js]: here too

**Review by rohanInsticator — COMMENTED** [review]
for fields which are integer only, we should add another validation to make sure the integer value passed in them is supported by ORTB on the backend.

File: modules/insticatorBidAdapter.js (hunk 1 above)
Comment (shubhamc-ins) [reply modules/insticatorBidAdapter.js]: Okay I'll update it

**Comment by shubhamc-ins** [comment]
> for fields which are integer only, we should add another validation to make sure the integer value passed in them is supported by ORTB on the backend.
```
  'minduration': I have added a check for minDuration <= maxDuration
  'maxduration': covered in above
  'protocols': should be a subarray of [2, 3, 5, 6, 7, 8]
  'startdelay': It can 
  'linearity': For now it should be `1` only
  'skip': either 0 or 1
  'skipmin': Should be fine with integer only
  'skipafter': Should be fine with integer only,
  'sequence': Should be fine with integer only,
  'battr': Subarray of ( [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17]),
  'maxextended': (value) => Should be fine with integer,
  'minbitrate': (value) => not sure, can you suggest here?,
  'maxbitrate': (value) => We have set this to 10000 default in ssp, but it can be higher, right?,
  'playbackmethod': (value) => Subarray ([1, 2, 3, 4]),
  'playbackend': (value) => Integer should be fine,
  'delivery': (value) => Subarray ([1, 2, 3]) according to ortb spec,
  'pos': (value) => should be integer between [0, 1, 2, 3, 4, 5, 6, 7],
  'api': (value) => Subarray([1, 2, 3, 4, 5, 6, 7]) according to ssp supported values,
```

**Review by rohanInsticator — APPROVED** [review]

**Pull request closed without merging**